.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,13 +23,14 @@ Steps to reproduce the behavior:
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+**Screenshots and/or execution output**
```

Comment:
by execution output do you mean a stack trace?

Reply:
Stack trace and/or whatever gets output from the execution